maxim/tests/openai/realtime/context_summarization_with_realtime_api_sdk.py

```
@@ -0,0 +1,459 @@
+        await asyncio.sleep(0.4 * attempts)
+        return await fetch_full_item(connection, item_id, state, attempts + 1)
+
+    # Done – remove the marker
+    state.waiting.pop(item_id, None)
+    return item
+
+
+# --------------------------------------------------------------------------- #
+# Realtime session                                                          #
+# --------------------------------------------------------------------------- #
+async def realtime_session(model="gpt-realtime", voice="shimmer", enable_playback=True):
```

Ruff (0.14.4)

Unused function argument: `voice`

(ARG001)